tests/unit/python/foglamp/services/core/service_registry/test_service_registry.py

```diff
@@ -104,7 +104,8 @@ def test_register_with_bad_management_port(self):
         assert excinfo.type is NonNumericPortError
         assert 0 == len(ServiceRegistry._registry)
 
-    def test_unregister(self):
+    @pytest.mark.asyncio
+    async def test_unregister(self):
```

Comment:
we are not passing any event_loop fixture to method

Reply:
restored old code.